Dockerfile.src

```diff
@@ -0,0 +1,13 @@
+FROM python:3.4-slim
```

Comment:
@josephharrington I have seen Dockerfile names with both dot and dash based separation. In some cases I came across Dockerfile name using both! But I agree more with the name you suggested to reduce confusion and to improve readability. Will fix that...